dandi/publish/models/contributor.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField, JSONField
+
+class Contributor(models.Model):
+    class RoleType(models.TextChoices):
+        Author = "Author"
+        Conceptualization = "Conceptualization"
+        ContactPerson = "ContactPerson"
+        DataCollector = "DataCollector"
+        DataCurator = "DataCurator"
+        DataManager = "DataManager"
+        FormalAnalysis = "FormalAnalysis"
+        FundingAcquisition = "FundingAcquisition"
+        Investigation = "Investigation"
+        Maintainer = "Maintainer"
+        Methodology = "Methodology"
+        Producer = "Producer"
+        ProjectLeader = "ProjectLeader"
+        ProjectManager = "ProjectManager"
+        ProjectMember = "ProjectMember"
+        ProjectAdministration = "ProjectAdministration"
+        Researcher = "Researcher"
+        Resources = "Resources"
+        Software = "Software"
+        Supervision = "Supervision"
+        Validation = "Validation"
+        Visualization = "Visualization"
+        Funder = "Funder"
+        Sponsor = "Sponsor"
+        StudyParticipant = "StudyParticipant"
+        Other = "Other"
+
+    name = models.CharField(default=str, max_length=150) # TODO: verify max_length is correct
+    email = models.TextField(default=str) # TODO: need max length, use charfield if possible
+    orcid = models.TextField(default=str) # TODO: same as email
```

Comment:
@mgrauer Do we have maximum lengths for contributor `name`, `email` and `orcid`?